components/connector-service/cmd/connectorservice/connectorservice.go

```diff
@@ -37,8 +39,13 @@ func main() {
 	certUtil := certificates.NewCertificateUtility()
 	tokenGenerator := tokens.NewTokenGenerator(options.tokenLength, tokenCache)
 
-	externalHandler := newExternalHandler(tokenCache, certUtil, tokenGenerator, options, env)
-	internalHandler := newInternalHandler(tokenCache, tokenGenerator, options.connectorServiceHost)
+	middlewares, appErr := monitoring.SetupMonitoring()
```

Comment:
Please consider renaming monitoring.SetupMonitoring. Something like monitoring.GetMiddlewares() would be more suitable as it returns a list of Middlewares